docs/dApps.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Decentralized applications (dApps)
+authors: Benjamin Pilia, Tim McMackin
+last_update:
+  date: 12 January 2024
+---
+
+import LucidDiagram from '@site/src/components/LucidDiagram';
+
+One of the main features of blockchains is _decentralization_: each transaction is verified by multiple nodes and its validation process does not rely on a single trusted third party.
+Decentralized applications (dApps or Dapps) take advantage of these features to create applications that are independent, transparent, and trustless.
+
+In general, dApps have these parts:
+
+- **Frontend**: An off-chain component that can act as a user interface to simplify interaction with the on-chain component, run off-chain processing, and get information from sources that are not available to the on-chain component
+- **Middleware**: Optionally, an [indexer](./developing/information/indexers) to interpret the backend information and provide it in a more convenient format for the front-end component
+- **Backend**: An on-chain component that consists of one or more [smart contracts](./smart-contracts)
+
+The off-chain component can be nearly any kind of program, including a web application, mobile or desktop app, or command-line interface.
+It relies on wallets and tools to interact with the smart contract on behalf of a user's Tezos account.
+
+<LucidDiagram width="640px" height="480px" src="https://lucid.app/documents/embedded/8caf9ef1-11e4-454a-bbb6-ef4852515959" id=".4NXymECcQqz" />
+
+Some of these tools that allow an off-chain component to interact with smart contracts include:
+
+- [Taquito](./dApps/taquito), an SDK for JavaScript/TypeScript applications
+- The [Tezos SDK for Unity](./unity), a toolkit for Unity applications
```

Comment:
Not everyone knows what Unity is, or for some people Unity is a Linux UI, so it would be better to:
- link "Unity" to unity.com
- mention the keyword "gaming" somehow, e.g. "gaming applications" or "gaming platform"

Reply:
Addressed in https://github.com/trilitech/tezos-developer-docs/pull/287